ext_emconf.php

```diff
@@ -11,10 +11,10 @@
     'title' => 'Paste reference for content elements',
     'description' => 'Paste reference instead of copy for content elements',
     'category' => 'plugin',
-    'version' => '3.0.0',
+    'version' => '4.0.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '12.4.0-12.4.99',
+            'typo3' => '13.4.0-13.5.99',
```

Comment:
According to the roadmap https://typo3.org/cms/roadmap the 13.4 will be the LTS release.

```suggestion
            'typo3' => '13.4.0-13.4.99',
```

Reply:
good, that you clarified the versions!